project_event/models/project_task.py

```diff
@@ -171,8 +174,10 @@ def request_reservation(self):
             'resource_type': self.resource_type,
             'room_id': self.room_id.id if self.room_id else None,
             'equipment_ids': [(4, self.equipment_id.id, 0)] if self.equipment_id else None,
+            'state': 'open'
         }
         new_event = calendar_event.create(values)
+        self.reservation_event_id = new_event.id
         if self.room_id:
```

Comment:
@eilst:  if self.room_id:
            self.reserve_equipment_inside(new_event.id
If this condition fails, how the application will behave or this condition execute always true